mindmeld/components/question_answerer.py

```diff
@@ -372,13 +373,35 @@ class NativeQuestionAnswerer(BaseQuestionAnswerer):
     """
 
     # a common resource loaded generally used across all resolvers during loading process
-    RESOURCE_LOADER = ResourceLoader.create_resource_loader(
-        app_path=None,
-        text_preparation_pipeline=TextPreparationPipelineFactory.create_text_preparation_pipeline(
-            language="en",
-            tokenizer=WhiteSpaceTokenizer()
-        )
-    )
+    # a class var because it is also used in the underlying `Indices` class
+    RESOURCE_LOADER = None
+
+    def __init__(self, *args, **kwargs):
+        """
+        Args:
+            resource_loader (ResourceLoader, optional): A resource loader object used by the
+                underlying entity resolver models
+        """
+        super().__init__(*args, **kwargs)
+
+        # update class' resource loader; use one if already passed-in
+        resource_loader = kwargs.get("resource_loader")
+        if not resource_loader:
+            text_preparation_pipeline = \
+                TextPreparationPipelineFactory.create_text_preparation_pipeline(
+                    language="en",
+                    tokenizer=WhiteSpaceTokenizer()
+                )
+            query_factory = QueryFactory.create_query_factory(
+                app_path=None,
+                text_preparation_pipeline=text_preparation_pipeline,
+                duckling=True
```

Comment:
@vrdn-23 @ritvikshrivastava  I think without explicitly setting `duckling=True`, the error message you see will still keep coming up if no resource_loaded is passed to NativeQA. Hence, instead of passing in a `text_preparation_pipeline` to `create_resource_loader`, I am passing a `query_factory` object directly.

Reply:
Yeah this is fine I think (otherwise we'll have to trickle this change down via resource_loader all the way to query factory)

Reply:
Curious as to what happens in the case when we aren't using duckling? In a system that doesn't have the duckling option/binary, what would happen by this statement?

Reply:
Good question. Made me think we shouldn't actually hard set duckling=True here. It is because QA component in MindMeld doesn't use "query" objects anywhere and hence no need a duckling recognizer.

Reply:
The previous behavior was creating a NoOpSystemEntityRecognizer. This change in flag kind of changes that.  
We should try to identify if there is a way to suppress that warning given in query_factory.py as it's misleading when creating a resource loader just for Native Question Answerer. Or look for an alternative route to creating a resource loader. Or make sure that the duckling flag is indeed trickled down to the Native QA

Reply:
If we're okay with a NoOpSystemEntityRecognizer for this use-case, then we can just pass ``system_entity_recognizer = NoOpSystemEntityRecognizer.get_instance()`` as a kwarg to create_query_factory. This would allow for passing the default recognizer while preventing the error message.

Reply:
```suggestion
                system_entity_recognizer = NoOpSystemEntityRecognizer.get_instance()
```

Reply:
Seems like a good alternative.